docker/images/rolling/Dockerfile

```diff
@@ -52,6 +52,7 @@ RUN apt-get update \
     tmux \
   && rm -rf /var/lib/apt/lists/*
 
+ENV PIP_BREAK_SYSTEM_PACKAGES 1
```

Comment:
By default, `pip` now won't allow system-wide installations. I wonder how the ROS folks will deal with this in `rosdep`.